### ISSUE TITLE
Is it possible to update only specific layers?

### ISSUE THREAD

**Issue by jihunchoi** [pfnet/chainer#1151]
Hello,
I love Chainer, especially its easy-to-use interfaces and extreme flexibility.
However, I couldn't find that if I can update only the weights of specific layers.
In other words, can I stop calculating the gradients at a certain stage?
The common scenario where this functionality is needed is: when using pre-trained embeddings and want to update only hidden layers' weights, not the embedding layers' weights.
Does Chainer have this feature, or at least is there any workaround for this?
Thank you.

**Comment by jihunchoi** [comment]
I don't know it is the intended usage, anyway I found a somewhat workaround of this.
In the above scenario,
1. Compute the hidden states, namely h_fixed, based on pre-trained embeddings, with volatile flag set to 'on'. (The computation history of this variable would not be used, so it is okay to set them as volatile)
2. Define non-volatile variable h, using h_fixed.data
3. Using h, continue further computations.
This will stop calculating gradients when the backpropagation arrives at h.

**Comment by iaroslav-ai** [comment]
+1

**Comment by felipecode** [comment]
Also, If I want to update not only specific layers but specific neurons as well.
Is it possible to set just  certain neurons as volatile ?
Thank you
